.coverage

```diff
@@ -1 +1 @@
 161, 2211, 165, 166, 2218, 2221, 2224, 2245, 203, 2424, 2258, 214, 2263, 240, 2089, 248, 254, 385, 279, 2436, 308, 394, 2382, 398, 351, 400, 2107, 356, 357, 358, 360, 2108, 364, 365, 2414, 2109, 369, 370, 373, 2422, 376, 2425, 378, 379, 380, 383, 384, 2112, 388, 2438, 2113, 2442, 395, 396, 2446, 399, 2448, 401, 405, 406, 407, 408, 2116, 413, 420, 424, 426, 440, 444, 2126, 470, 472, 2468, 474, 477, 478, 479, 480, 485, 486, 488, 2538, 2546, 2547, 500, 2550, 2552, 541, 554, 563, 564, 2145, 586, 599, 606, 607, 608, 611, 617, 618, 619, 620, 621, 624, 635, 643, 646, 671, 693, 731, 469, 781, 367, 800, 834, 853, 858, 868, 890, 899, 908, 917, 940, 498, 499, 951, 986, 1002, 170, 1022, 1038, 1040, 1041, 1042, 1053, 1058, 1065, 1139, 1191, 1195, 1203, 1204, 1205, 1206, 1214, 1216, 1219, 1221, 1222, 1224, 1228, 1238, 1239, 1240, 1241, 1242, 1243, 1244, 1245, 1246, 1247, 1248, 1251, 1264, 1265, 1267, 1270, 1284, 2262, 1289, 1291, 1292, 1294, 1295, 1296, 1297, 1306, 1307, 1308, 1313, 1315, 1317, 1320, 1321, 1324, 1348, 1350, 1351, 1352, 1353, 2110, 1474, 1498, 1505, 1517, 1518, 1519, 1523, 1524, 1528, 1529, 1530, 1552, 1553, 1285, 1602, 1603, 2111, 1670, 1671, 1672, 1673, 1677, 1704, 1706, 1717, 1731, 1742, 1747, 1773, 1786, 1791, 1792, 1795, 1804, 1810, 1811, 1812, 1813, 1814, 1817, 1830, 1832, 1834, 1835, 1836, 1837, 1840, 1846, 1847, 1848, 1849, 1850, 1851, 1855, 1863, 1864, 1865, 1868, 1870, 1871, 1872, 1873, 1874, 1877, 1878, 1879, 1880, 1881, 1884, 1885, 1886, 1887, 1888, 1889, 1890, 1891, 1892, 1893, 1894, 1895, 1896, 1897, 1898, 1899, 1900, 1904, 1912, 1919, 1929, 1930, 1931, 1932, 1937, 1960, 1961, 1967, 1990, 1991, 1992, 2003, 2013, 2014, 2025, 2026, 2030, 2034, 2043], "/home/andela/andela/andelasims/cp1-amity/tests/person.py": [1, 2, 3, 5, 8, 11], "/usr/local/lib/python2.7/dist-packages/pbr/version.py": [19, 21, 22, 23, 25, 28, 29, 30, 31, 36, 40, 43, 44, 56, 57, 58, 59, 60, 61, 63, 65, 70, 73, 96, 106, 109, 112, 115, 118, 121, 146, 147, 151, 154, 156, 158, 159, 160, 162, 174, 175, 176, 177, 178, 179, 180, 182, 190, 191, 197, 198, 199, 200, 204, 208, 213, 225, 226, 227, 228, 234, 236, 238, 240, 248, 277, 319, 327, 332, 333, 337, 344, 346, 351, 353, 363, 372, 388, 389, 400, 404, 405, 406, 409, 411, 417, 418, 419, 420, 422, 426, 431, 438, 439, 440, 441, 448, 450, 457, 459, 460, 461, 463, 468, 469, 471], "/usr/local/lib/python2.7/dist-packages/funcsigs/__init__.py": [6, 7, 8, 9, 10, 11, 13, 14, 656, 18, 451, 20, 277, 23, 24, 281, 26, 27, 28, 285, 31, 289, 411, 293, 294, 39, 801, 794, 455, 626, 486, 659, 596, 446, 55, 442, 315, 190, 191, 339, 194, 195, 198, 199, 200, 201, 202, 343, 204, 207, 211, 212, 213, 214, 215, 787, 603, 218, 347, 476, 478, 607, 480, 481, 354, 483, 485, 358, 236, 238, 240, 241, 242, 243, 244, 373, 246, 375, 248, 249, 379, 523, 622, 383], "/home/andela/andela/andelasims/cp1-amity/tests/__init__.py": [1], "/usr/local/lib/python2.7/dist-packages/funcsigs/version.py": [1], "/usr/local/lib/python2.7/dist-packages/pbr/__init__.py": [1]}}
```

Comment:
This file should not be committed add it  to your gitignore file and remove it from git